## Respuesta 5

Para resolver esto se aborda 2 partes:

### 1. Mencionar supuestos del modelo de Regresion Lineal

- **Linealidad**: La relación entre la variable dependiente y las variables independientes es lineal.
- **Independencia**: Las observaciones son independientes entre sí.
- **Homoscedasticidad**: La varianza de los errores es constante para todos los valores de las variables independientes.
- **Normalidad**: Los errores se distribuyen normalmente con media cero y varianza constante.
- **No multicolinealidad**: Las variables independientes no están perfectamente correlacionadas entre sí.

### 2. Calculo de los estimadores del modelo

El modelo se representa como:

𝐘 = 𝐗 β + *e*

Donde:

- **Y** es el vector de observaciones.
- **X** es la matriz de diseño de las variables independientes.
- **β** es el vector de coeficientes que queremos estimar.
- ***e*** es el vector de errores, que se asume que sigue una distribución normal con media cero y varianza constante.

Para estimar β se utiiza la formula de los minimos cuadrados ordinarios (OLS)

β^ = (XᵀX)\^-1 XᵀY


Dado:

In [1]:
import numpy as np

# Definir los datos
Y = np.array([6, 4, 23, 5, 11])
X = np.array([
    [3, 4, 5, 6, 13],
    [10, 3, 7, 1, 4],
    [10, 25, 12, 14, 16],
    [7, 14, 20, 23, 7],
    [3, 4, 4, 4, 10]
])

**Paso 1:** Primero se debe cacular la transpuesta de X

In [2]:
# Transponer la matriz X
X_T = X.T

**Paso 2:** Multiplicar Xᵀ por X

In [3]:
# Multiplicar X_T por X
X_T_X = np.dot(X_T, X)

**Paso 3:** Multiplicar Xᵀ por Y

In [4]:
# Multiplicar X_T por Y
X_T_Y = np.dot(X_T, Y)

**Paso 4**: Calcular la inversa de XᵀX

In [5]:
# Calcular la inversa de X_T_X
X_T_X_inv = np.linalg.inv(X_T_X)

**Paso 5:** Multiplicar la inversa por XᵀY para obtener β^

In [6]:
# Calcular los coeficientes beta
beta_hat = np.dot(X_T_X_inv, X_T_Y)
beta_hat

array([-11.24060403,   4.02248322,  15.7511745 , -13.04916107,
         1.78238255])

#### Resultado:
Los coeficientes estimados para β^ son:


β^ = \begin{pmatrix} -11.24060403 \\ 4.02248322 \\ 15.7511745 \\ -13.04916107 \\ 1.78238255 \end{pmatrix}